narwhal/executor/src/subscriber.rs

```diff
@@ -152,7 +152,7 @@ async fn create_and_run_subscriber(
 
 impl<Network: SubscriberNetwork> Subscriber<Network> {
     /// Returns the max amount of pending consensus messages we should expect.
```

Comment:
Maybe update the comment to `Returns the max number of sub-dag to fetch payloads concurrently`.